libr/core/cmd_search.c

```diff
@@ -230,6 +231,8 @@ R_API int r_core_search_prelude(RCore *core, ut64 from, ut64 to, const ut8 *buf,
 	int ret;
 	ut64 at;
 	ut8 *b = (ut8 *)malloc (core->blocksize);
+	int timeout = r_config_get_i (core->config, "anal.aaptimeout");
```

Comment:
if timeout is < 1 we should not get the time or check it at all, otherwise we have a performance problem